### DISCUSSION TITLE
Does sdmTMB support areal data?

### DISCUSSION THREAD

**Discussion by ericward-noaa** [sdmTMB/sdmTMB#126] [Q&A]
Question via email:

_Can I use the code for areal panel data also? As in counties? It is not clear where we add the dataset to the model._

**Comment by ericward-noaa** [comment]
sdmTMB does not yet support the use of areal data. There is some working models over on the CAR branch here, 
https://github.com/pbs-assess/sdmTMB/tree/CAR

These have not been as thoroughly simulation tested as other features though -- and have not been updated to include newer features (delta models, etc.)

**Comment by seananderson** [comment]
Also, you can fit areal data with sdmTMB by taking the centroids of each grid cell or polygon and fitting them as a geostatistical model. I presume this will be less computationally efficient than something like a CAR model, but does have advantages in terms of interpretation of the spatial covariance parameters. See https://doi.org/10.1016/S0378-3758(03)00111-3
